src/core/textures/Texture.js

```diff
@@ -258,6 +280,24 @@ export default class Texture extends EventEmitter
     }
 
     /**
+     * returns uvs
+     * @member PIXI.TextureUvs#
+     * @memberof PIXI.Texture#
+     */
+    get uvs()
+    {
+        return this._uvs;
+    }
+
+    /**
+     * Updates uvs accordingly to the texture
+     */
+    updateUvs()
+    {
+        this._updateUvs();
+    }
+
+    /**
      * Helper function that creates a Texture object from the given image url.
      * If the image is not in the texture cache it will be  created and loaded.
      *
```

Comment:
I do not think we should add these new properties to the texture object. Mainly as they are VERY specific to certain non-core elements of Pixi. How about we add these properties to the texture transform class?